bundles/framework/mydata/view/MyViews/MyViewsTab.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MyViewsList } from './MyViewsList';
+import { Button, Message } from 'oskari-ui';
+import styled from 'styled-components';
+
+const BUNDLE_NAME = 'PersonalData';
+
+const ButtonContainer = styled.div`
+    margin: 10px 0 10px 0;
+    display: flex;
+    justify-content: flex-end;
+`;
+
+export const MyViewsTab = ({ controller, state }) => {
+
+    return (
+        <>
```

Comment:
Use `<React.Fragment>` instead of `<>`. It's a convention we've used that's probably not documented anywhere.